### PR TITLE
Some documentation about how to use regtest with defiads

### PR DESCRIPTION
**Pull request by philbertw4**
Per conversation with @notmandatory, switching this over to a separate pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,18 @@ negotiation and thereby give rise to bitcoin's first truly risk less interest ra
 $ cargo update
 $ cargo build --release
 ```
-# Run
+
+# Running On Local Regtest Network (Recommended for developers)
```
Comment (notmandatory) [inline README.md]: I don't think it's necessary to include "(Recommended for developers)", anyone doing development should be familiar with it.

File: README.md [hunk 2]
```diff
@@ -60,7 +60,18 @@ negotiation and thereby give rise to bitcoin's first truly risk less interest ra
...
+1. Be sure to have a somewhat recent version of bitcoin core installed. Importantly, it must be a version which serves BIP 157 blockfilters.
```
Comment (notmandatory) [inline README.md]: If you want to include a link, [this is the BIP 157 blockfilters branch](https://github.com/jimpo/bitcoin/tree/bip157-net) associated with the BIP.

File: README.md (hunk 1 above)
Comment (philbertw4) [reply README.md]: Thanks. Updated.

File: README.md (hunk 2 above)
Comment (philbertw4) [reply README.md]: Links included. Thanks.

**Comment by ZmnSCPxj** [comment]
Suggest rebase on top of current master (which would remove the first commit as well, since a version of it has been merged in).

**Comment by philbertw4** [comment]
Ok, done (I think). There will need to be more documentation for developers at some point, but this should at least help people who are less familiar with the various tools to get up and running at least slightly quicker.

**Comment by philbertw4** [comment]
Since this is just some minor doc fixes, are there any objections to me merging and closing this?

**Pull request merged**